main.go

```diff
@@ -87,7 +87,8 @@ func init() {
 	pflag.String("http-proto", "https", "Set to 'http' if necessary")
 	pflag.String("http-user", "", "HTTP username for pulling the remote file")
 	pflag.String("http-pass", "", "HTTP password for pulling the remote file")
-
+	pflag.String("http-header-name", "", "HTTP header name")
+	pflag.String("http-header-value", "", "HTTP header value")
```

Comment:
nit: Let's fill it with context. "HTTP header value for pulling the remote file"